Orchard1/Restart-Site/Restart-Site.psm1

```diff
@@ -33,13 +33,14 @@ function Restart-Site
         }
 
         # Stopping IIS site and app pool.
-        Stop-IISSite $SiteName -confirm:$false
+        Stop-IISSite $SiteName -Confirm:$false
```

Comment:
I have never seen a PS argument used like this.
Wouldn't that be the same as `Stop-IISSite $SiteName -Confirm $false`?
Also [according to the docs](https://learn.microsoft.com/en-us/powershell/module/iisadministration/stop-iissite?view=windowsserver2022-ps#-confirm) Confirm's default value is already false, so you can just write `Stop-IISSite $SiteName`

Reply:
I have, because I had to... :)
`Stop-IISSite $SiteName -Confirm $false` doesn't work (at least in Windows PS, I didn't try PowerShell Core).
I thought so too, but it was added like this in 2016 (probably because it was necessary) and it still is - if you don't set that switch to false, then you are prompted for confirmation.

Reply:
I've tested it and you are right. Looks like their documentation is wrong :(